recipes/master.rb

```diff
@@ -36,6 +36,11 @@
   not_if      { ::File.exist?("#{node['kube-hops']['pki']['dir']}/apiserver-kubelet-client.crt") }
 end
 
+master_hostname = node['hostname']
```

Comment:
Why not directly the FQDN? This won't work on GCP, see https://logicalclocks.atlassian.net/browse/HOPSWORKS-1739

Reply:
I am setting FQDN on centos and hostname on ubuntu because for some reason ubuntu does not work with the fqdn. I need to check if it is still the case after your changes.